AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/adaptors-toc.xml

```diff
@@ -7,11 +7,19 @@
                 <tocitem text="Extend From Pajek File" target="au.gov.asd.tac.constellation.views.dataaccess.adaptors.plugins.extend.ExtendFromPajekPlugin"/>
                 <tocitem text="Extend From GraphML File" target="au.gov.asd.tac.constellation.views.dataaccess.adaptors.plugins.extend.ExtendFromGraphMLPlugin"/>
                 <tocitem text="Extend From GML File" target="au.gov.asd.tac.constellation.views.dataaccess.adaptors.plugins.extend.ExtendFromGMLPlugin"/>
+                <tocitem text="Extend From GDELT" target="au.gov.asd.tac.constellation.views.dataaccess.adaptors.plugins.extend.ExtendFromGDELTPlugin"/>
```

Comment:
Could this TOC item text be "Extend From GDELT Knowledge Graph" to match the name displayed in Data Access View and the title of the help page?